src/xdocs/config_javadoc.xml

```diff
@@ -967,6 +967,145 @@ public Runnable printLater(String s) {
       </subsection>
     </section>
 
+    <section name="JavadocMissingLeadingAsterisk">
+      <p>Since Checkstyle 8.38</p>
+      <subsection name="Description" id="JavadocMissingLeadingAsterisk_Description">
+        <p>
+          Checks if the javadoc has
+          <a href="https://docs.oracle.com/en/java/javase/14/docs/specs/javadoc/doc-comment-spec.html#leading-asterisks">
+            leading asterisks
+          </a>
+          on each line.
+        </p>
+        <p>
+          The check does not require asterisks on the first line, nor on the last line
+          if it is blank. All other lines in a Javadoc should start with <code>*</code>,
```

Comment:
What is meant by first and last line?
First line is always `/**` which I technically understand as the leading asterisk as it aligns with the next lines asterisk. Similar to the last line. Is it referring to those asterisks are not considered leading by our parser?

Reply:
The specification is very unclear in this case. Javadoc tool removes initial asterisks even in such comments:
```java
/** * First line.
 * Second line.
 * */ // Last line
```
becomes "First line. Second line.", but no one writes Javadoc comments this way.